CONTRIBUTING.md

```diff
@@ -74,3 +74,45 @@ To ensure you don't need to worry about formatting and linting when contributing
 - Integration in your editor:
     - For [Black](https://black.readthedocs.io/en/stable/integrations/editors.html)
     - For other integrations please look at the documentation for your editor
+
+### Developement environment on Mac with Apple Silicon M1 processor (arm64 architecture)
```

Comment:
Thanks for the explanations 👍 
Maybe instead of repeating them in both CONTRIBUTING and INSTALL, you can perhaps only leave them in INSTALL, and then link to the relevant section from the CONTRIBUTING file - that will be slightly more concise.